CHANGES_NEXT_RELEASE

```diff
@@ -4,3 +4,26 @@ Fix: check QoS option for MQTT commands
 Feature: use AMQP message handler, add reconnections and error handlers
 Feature: add and use AMQP config env vars (#268)
 Fix: upgrade mqtt dep from 1.14.1 to 2.18.8
+Fix: use PM2 in Dockerfile (#336)
```

Comment:
Number 336 was meaninfull in iota-json repository. Just 

```
Fix: use PM2 in Dockerfile
```

would be enough here.

Reply:
Fixed with ba73fd8cbc49d20264b05bfbca1de4b332a28ee8